### PR TITLE
HUD section in Your first game update to 3.4

### PR DESCRIPTION
**Pull request by DiegoCCMM**
Update HUD from `Custom fonts` -> `Theme Overrides/Fonts` and the images that are used to illustrate that.

### CODE REVIEW

**Comment by DiegoCCMM** [comment]
I've also added a PAT note for beginners in GitHub

File: community/contributing/pr_workflow.rst [hunk 1]
```diff
@@ -351,6 +351,10 @@ Git will ask you for your username and password, and the changes will be sent
 to your remote. If you check the fork's page on GitHub, you should see a new
 branch with your added commits.
 
+.. note:: Since july 2020, GitHub changed the way you verify yourself in HTTPS clones.
+          The password you must introduce is a `PAT
+https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token`_
```
Comment (Calinou) [inline community/contributing/pr_workflow.rst:356]: Doesn't GitHub send you a message when trying to push to an HTTPS remote to warn you about this? I feel we should avoid duplicating their documentation.

Comment (DiegoCCMM) [reply community/contributing/pr_workflow.rst:356]: Absolutely right, should i revert it to keep the other commit?

Comment (Calinou) [reply community/contributing/pr_workflow.rst:356]: @DiegoCCMM Yes, I think it's better to revert this part for now.

**Comment by mhilbrunner** [comment]
Hey, our CI is fixed now for the 3.4 branch - could you please rebase/push to get a new build? Thanks :)

**Comment by akien-mga** [comment]
Superseded by #5542.

**Pull request closed without merging**